src/Makefile

```diff
@@ -12,8 +12,9 @@ CXX_TARGETS := \
 
 TARGETS := $(C_TARGETS) $(CXX_TARGETS)
 
-CFLAGS := -O2 -Wall -Wshadow
-CXXFLAGS := -O2 -std=c++11 -Wall -Wextra -Wshadow -Wno-sign-compare -Werror
+override CFLAGS   := -O2 -Wall -Wshadow $(CFLAGS)
+override CXXFLAGS := -O2 -std=c++11 -Wall -Wextra -Wshadow -Wno-sign-compare \
+		     -Werror $(CXXFLAGS)
```

Comment:
According to the [GNU make documentation](https://www.gnu.org/software/make/manual/html_node/Override-Directive.html), this means that the flags passed on the command line won't be used at all. Should this be `+=` instead of `:=`?

Reply:
You're right, I missed the `$(CFLAGS)` at the end.